iroh-util/src/exitcodes.rs

```diff
@@ -0,0 +1,16 @@
+//! iroh exit codes
+//!
+//! Exit code constants intended to be passed to
+//! `std::process::exit()`
```

Comment:
Just a note: in the `iroh` CLI, clap does the exiting itself. I know it does a non-zero exit code when things are wrong, and since we don't have more information about errors at that level at the moment that's good enough for now.